### PR TITLE
add room template, use jinja templates

### PR DESCRIPTION
**Pull request by DanLindeman**

### CODE REVIEW

File: kcards/app.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def configure_logging(app):
 
 
 def register_blueprints(app):
+    app.register_blueprint(views.room.blueprint)
```
Comment (DanLindeman) [inline kcards/app.py]: Probably minimally by api and front facing maybe in `register_frontend()` and `register_backend()`?

File: kcards/templates/kcards.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!doctype html PUBLIC "-//IETF//DTD HTML 2.0//EN">
```
Comment (jacebrowning) [inline kcards/templates/kcards.html]: Is `"-//IETF//DTD HTML 2.0//EN` correct? I don't know much about this...

File: kcards/views/room.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from flask import Blueprint, Response, render_template
+
+
+blueprint = Blueprint('room', __name__)
+
+
+@blueprint.route("/rooms/<key>")
+def detail(key):
+    # Placeholder for how to 404 when a room is not in the room list.
+    if key is 'not_found':
```
Comment (jacebrowning) [inline kcards/views/room.py]: Should be `key == 'not_found'`.

File: kcards/views/room.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        key=None
+    return Response(render_template("room.html", key=key))
```
Comment (jacebrowning) [inline kcards/views/room.py]: We should have newlines at the end of files.

File: kcards/templates/kcards.html (hunk 2 above)
Comment (DanLindeman) [inline kcards/templates/kcards.html]: Removing this seems to have no effect as of yet. I have no idea what it does, copy-pasta'd from something I found online. Seems removable.

**Pull request merged**